syntax = "proto3";
// 설계도
// input: bytes frame(image), output: int32
service RemoteControlService {
    rpc process(UserRequest) returns (ServerReply) {}
}

// UserRequest
message UserRequest {
    bytes img_bytes = 1;
    int32 width = 2;
    int32 height = 3;
    int32 channel = 4;
}

// ServerReply
message ServerReply {
    int32 distance = 1;
    bool is_abscent = 2;
}